### ISSUE TITLE
Issues building on Ubuntu 22.04

### ISSUE THREAD

**Issue by loood** [htrefil/rkvm#41]
Building this in a ubuntu:jammy docker image. After running `apt-get update ; apt-get -y install cargo clang libevdev-dev build-essential pkg-config` I get this error. Any ideas?
```
   Compiling futures-executor v0.3.28
   Compiling futures v0.3.28
   Compiling inotify v0.10.0
   Compiling tokio-rustls v0.24.0
   Compiling smallvec v1.10.0
   Compiling bincode v1.3.3
   Compiling toml v0.5.11
error[E0531]: cannot find unit struct, unit variant or constant `ABS_PROFILE` in module `glue`
  --> rkvm-input/src/abs.rs:84:19
   |
84 |             glue::ABS_PROFILE => Self::Profile,
   |                   ^^^^^^^^^^^ not found in `glue`

error[E0425]: cannot find value `ABS_PROFILE` in module `glue`
   --> rkvm-input/src/abs.rs:133:36
    |
133 |             Self::Profile => glue::ABS_PROFILE,
    |                                    ^^^^^^^^^^^ not found in `glue`

Some errors have detailed explanations: E0425, E0531.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `rkvm-input` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
root@016e282a05bf:/workspace# \
```

**Comment by loood** [comment]
I should note this is from the latest commit. tags/0.3.3 compiled fine.

**Comment by htrefil** [comment]
The master branch adds support for ABS_* events. It looks like this ABS_PROFILE is not defined in your kernel. We'll need to add compile time detection in build.rs and disable it if it's not present.

**Comment by knotu** [comment]
While it's not my main install, I also have this issue in Void Linux. I can keep everything on on 0.3.3 for now. rkvm has worked great for the months I've been using it. If not for this, I wouldn't have even bothered using/moving to wayland. So I appreciate all the work you've done.

**Comment by htrefil** [comment]
Fixed in 0.4.1 🎉
